lib/Sema/CSDiagnostics.cpp

```diff
@@ -3424,6 +3427,37 @@ bool MissingMemberFailure::diagnoseForDynamicCallable() const {
   return false;
 }
 
+bool MissingMemberFailure::diagnoseInLiteralCollectionContext() const {
+  auto &cs = getConstraintSystem();
+  auto *expr = castToExpr(getAnchor());
+  auto *parentExpr = cs.getParentExpr(expr);
+  auto &solution = getSolution();
+
+  if (!(parentExpr && isa<UnresolvedMemberExpr>(expr)))
+    return false;
+
+  auto parentType = getType(parentExpr);
+
+  if (!cs.isCollectionType(parentType) && !parentType->is<TupleType>())
+    return false;
+
+  if (isa<TupleExpr>(parentExpr)) {
+    parentExpr = cs.getParentExpr(parentExpr);
```

Comment:
I think this has to be `findParentExpr` as well :/

Reply:
Fixed!